iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuiz/Views/Modals/ProblemsLimitReachedModalViewControllerDelegate.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+protocol ProblemsLimitReachedModalViewControllerDelegate: AnyObject {
```

Comment:
Not necessary to separate the delegate and view the controller declaration, until the delegate becomes too large. So, you can move this delegate protocol directly to the `ProblemsLimitReachedModalViewController.swift`